### PR TITLE
Arb-ts updates required for arb-token-list

### PR DESCRIPTION
**Pull request by yahgwai**

### CODE REVIEW

File: packages/arb-ts/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "postinstall": "$npm_execpath run gen:abi",
     "prepublishOnly": "yarn build && yarn format",
     "preversion": "yarn lint",
-    "build": "./scripts/builder",
+    "build": "./scripts/builder && cp -R ./src/lib/abi/*.d.ts ./dist/lib/abi",
```
Comment (DZGoldman) [inline packages/arb-ts/package.json:22]: why is this necessary?

Comment (yahgwai) [reply packages/arb-ts/package.json:22]: Apparently tsc doesn't copy d.ts files, it only creates them. https://stackoverflow.com/questions/56018167/typescript-does-not-copy-d-ts-files-to-build

So I'm manually copying them as part of the build
